core/tests/global-teardown.js

```diff
@@ -0,0 +1,60 @@
+import fs from 'fs';
+import { CDPClient } from 'monocart-coverage-reports';
+import { addCoverageReport } from 'monocart-reporter';
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const globalTeardown = async (config) => {
+  if (!process.env.CI) {
```

Comment:
This is to prevent collecting coverage on CI as we the source file instrumentation is different on vercel builds which will be handled in a follow up PR.